<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"><class>IDD_PROPPAGE_COMPARE_FOLDER</class>
<widget class="QDialog" name="IDD_PROPPAGE_COMPARE_FOLDER">
<property name="geometry">
<rect><x>0</x><y>0</y>
<width>510</width><height>410</height></rect></property>
<property name="windowTitle"><string>Folder compare</string></property>
<widget class="QLabel" name="IDC_STATIC">
<property name="geometry">
<rect><x>14</x><y>12</y><width>200</width><height>20</height></rect></property>
<property name="text"><string>&amp;Compare method:</string></property>
<property name="inputMethodHints"><set>Qt::ImhMultiLine</set></property><property name="wordWrap"><bool>true</bool></property></widget>
<widget class="QComboBox" name="IDC_COMPAREMETHODCOMBO">
<property name="geometry">
<rect><x>260</x><y>12</y><width>228</width><height>22</height></rect></property>
<property name="sizeAdjustPolicy"><enum>QComboBox::AdjustToContents</enum></property></widget>
<widget class="QCheckBox" name="IDC_COMPARE_STOPFIRST">
<property name="geometry">
<rect><x>14</x><y>48</y><width>404</width><height>20</height></rect></property>
<property name="text"><string>S&amp;top after first difference</string></property>
</widget>
<widget class="QCheckBox" name="IDC_IGNORE_SMALLTIMEDIFF">
<property name="geometry">
<rect><x>14</x><y>72</y><width>444</width><height>20</height></rect></property>
<property name="text"><string>Ign&amp;ore time differences less than 3 seconds</string></property>
</widget>
<widget class="QCheckBox" name="IDC_COMPARE_WALKSUBDIRS">
<property name="geometry">
<rect><x>14</x><y>96</y><width>404</width><height>20</height></rect></property>
<property name="text"><string>&amp;Include unique subfolders contents</string></property>
</widget>
<widget class="QCheckBox" name="IDC_RECURS_CHECK">
<property name="geometry">
<rect><x>14</x><y>120</y><width>404</width><height>20</height></rect></property>
<property name="text"><string>&amp;Include Subfolders</string></property>
</widget>
<widget class="QCheckBox" name="IDC_EXPAND_SUBDIRS">
<property name="geometry">
<rect><x>14</x><y>144</y><width>404</width><height>20</height></rect></property>
<property name="text"><string>&amp;Automatically expand all subfolders</string></property>
</widget>
<widget class="QCheckBox" name="IDC_IGNORE_REPARSEPOINTS">
<property name="geometry">
<rect><x>14</x><y>168</y><width>404</width><height>20</height></rect></property>
<property name="text"><string>Ignore &amp;Reparse Points</string></property>
</widget>
<widget class="QLabel" name="IDC_STATIC">
<property name="geometry">
<rect><x>14</x><y>192</y><width>400</width><height>20</height></rect></property>
<property name="text"><string>&amp;Quick compare limit (MB):</string></property>
<property name="inputMethodHints"><set>Qt::ImhMultiLine</set></property><property name="wordWrap"><bool>true</bool></property></widget>
<widget class="QTextEdit" name="IDC_COMPARE_QUICKC_LIMIT">
<property name="geometry">
<rect><x>14</x><y>216</y><width>60</width><height>28</height></rect></property>
<property name="verticalScrollBarPolicy"><enum>Qt::ScrollBarAlwaysOff</enum></property><property name="horizontalScrollBarPolicy"><enum>Qt::ScrollBarAlwaysOff</enum></property><property name="frameShape"><enum>QFrame::NoFrame</enum></property></widget>
<widget class="QLabel" name="IDC_STATIC">
<property name="geometry">
<rect><x>14</x><y>252</y><width>400</width><height>20</height></rect></property>
<property name="text"><string>&amp;Binary compare limit (MB):</string></property>
<property name="inputMethodHints"><set>Qt::ImhMultiLine</set></property><property name="wordWrap"><bool>true</bool></property></widget>
<widget class="QTextEdit" name="IDC_COMPARE_BINARYC_LIMIT">
<property name="geometry">
<rect><x>14</x><y>276</y><width>60</width><height>28</height></rect></property>
<property name="verticalScrollBarPolicy"><enum>Qt::ScrollBarAlwaysOff</enum></property><property name="horizontalScrollBarPolicy"><enum>Qt::ScrollBarAlwaysOff</enum></property><property name="frameShape"><enum>QFrame::NoFrame</enum></property></widget>
<widget class="QLabel" name="IDC_STATIC">
<property name="geometry">
<rect><x>14</x><y>308</y><width>460</width><height>60</height></rect></property>
<property name="text"><string>&amp;Number of compare threads (a negative value implies addition of the number of available CPU cores):</string></property>
<property name="inputMethodHints"><set>Qt::ImhMultiLine</set></property><property name="wordWrap"><bool>true</bool></property></widget>
<widget class="QTextEdit" name="IDC_COMPARE_THREAD_COUNT">
<property name="geometry">
<rect><x>14</x><y>372</y><width>60</width><height>28</height></rect></property>
<property name="verticalScrollBarPolicy"><enum>Qt::ScrollBarAlwaysOff</enum></property><property name="horizontalScrollBarPolicy"><enum>Qt::ScrollBarAlwaysOff</enum></property><property name="frameShape"><enum>QFrame::NoFrame</enum></property></widget>
<widget class="QPushButton" name="IDC_COMPAREFOLDER_DEFAULTS">
<property name="geometry">
<rect><x>14</x><y>416</y><width>140</width><height>28</height></rect></property>
<property name="text"><string>Defaults</string></property>
</widget>
</widget><resources/><connections/></ui>